recipe/meta.yaml

```diff
@@ -10,14 +10,16 @@ source:
 
 build:
   number: 1
-  script: python setup.py install
+  script: python -m pip install --no-deps --ignore-installed .
   skip: True  # [py>=35]
 
 requirements:
   build:
     - python
+    - pip
```

Comment:
I think you need backports package in host and run to avoid backports/__init__.py from being pacakaged by each and every backports package.

Reply:
Yeah, that was my original thought, and then I saw this issue: https://bitbucket.org/brandon/backports.ssl_match_hostname/issues/11/setuppy-breaks-other-backports-packages

But I guess that is a problem with pip and this package.

Reply:
> But I guess that is a problem with pip and this package.

That sounds right. Don't think that issue affects us.